### PR TITLE
Update v2

### PR DESCRIPTION
**Pull request by jspeed-meyers**
@rob, this is an attempt to merge back in the progress from the private fork into the public fork. Does it look right to you? Can you please take a look and let me know?

JS

### CODE REVIEW

**Comment by jspeed-meyers** [comment]
@mchadwick-iqt, does this look right to you? Of course, this operation is not as simple as I would like it to be.

**Review by rcaudill — APPROVED** [review]
@jspeed-meyers this looks correct to me!

**Comment by mchadwick-iqt** [comment]
@jspeed-meyers - I pushed a new commit to remove a number of obsolete files that had been retained with the merge.  Can you do a quick once-over and make sure we're still good from your perspective?

**Comment by jspeed-meyers** [comment]
@mchadwick-iqt, It looks good to me. I am going to look at it one more time with fresh eyes tomorrow morning and then I'll merge.

File: axis-ptz/docker-compose.axis-ptz.devcontainer.yml [hunk 1]
```diff
@@ -4,5 +4,5 @@ services:
   axis-ptz:
     command: /bin/sh -c "while sleep 1000; do :; done"
     volumes:
-      - /home/pi/Projects/SkyScan/axis-ptz:/workspace:cached # Additional bind mount
+      - /home/pi/Projects/SkyScan-Private/axis-ptz:/workspace:cached # Additional bind mount
```
Comment (jspeed-meyers) [inline axis-ptz/docker-compose.axis-ptz.devcontainer.yml:7]: @rcaudill, is this use of a different volume path a problem?

**Pull request merged**